docs/configuration/layers/cluster-option.md

```diff
@@ -0,0 +1,20 @@
+###### [SMK](../../..) / [Configuration](..) / [Layers](.)
+
+# Layer Cluster Option Objects
+
+This object defines properties that influence the rendering of clusters of points.
+
+A clusterOption object contains these properties, which have these default values. 
```

Comment:
Maybe follow some of the existing pattern for other objects? eg. https://bcgov.github.io/smk/docs/configuration/layers/vector

"This is the default configuration for the clusterOption object."

Reply:
I think the pattern "This is the default configuration for ..." is applied for docs on layers. Besides "vector", "wms" and "esri-dynamic" follow it, for example, "This is default configuration for the WMS layer".

I was following the pattern used by "style" and "attribute": "This object defines ...". Since clusterOption is a new object, I think this pattern is appropriate. Let me know if you disagree.

Reply:
I see. The language just sounds awkward to me. I prefer more 'natural language' in documentation. Change if you want or leave it as is, I'll leave it up to you.